workflows/noxfile.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2021 Google LLC
```

Comment:
Why do you want to have this file? Can you remove this, or use the current `noxfile-template.py`?

Reply:
Removed.